lib/workers/miq_worker_types.rb

```diff
@@ -1,77 +1,77 @@
+  "ManageIQ::Providers::Openstack::InfraManager::MetricsCollectorWorker"      => [:openstack],
+  "ManageIQ::Providers::Openstack::InfraManager::RefreshWorker"               => [:openstack],
+  "ManageIQ::Providers::Openstack::NetworkManager::EventCatcher"              => [:openstack],
+  "ManageIQ::Providers::Openstack::NetworkManager::MetricsCollectorWorker"    => [:openstack],
+  "ManageIQ::Providers::Openstack::NetworkManager::RefreshWorker"             => [:openstack],
+  "ManageIQ::Providers::Redhat::InfraManager::EventCatcher"                   => [:ovirt],
+  "ManageIQ::Providers::Redhat::InfraManager::MetricsCollectorWorker"         => [:ovirt],
+  "ManageIQ::Providers::Redhat::InfraManager::RefreshWorker"                  => [:ovirt],
+  "ManageIQ::Providers::StorageManager::CinderManager::EventCatcher"          => [:openstack],
+  "ManageIQ::Providers::StorageManager::CinderManager::RefreshWorker"         => [:openstack],
+  "ManageIQ::Providers::StorageManager::SwiftManager::RefreshWorker"          => [:openstack],
+  "ManageIQ::Providers::Vmware::CloudManager::EventCatcher"                   => [:vmware],
+  "ManageIQ::Providers::Vmware::CloudManager::RefreshWorker"                  => [:vmware],
+  "ManageIQ::Providers::Vmware::InfraManager::EventCatcher"                   => [:vmware],
+  "ManageIQ::Providers::Vmware::InfraManager::MetricsCollectorWorker"         => [:vmware],
+  "ManageIQ::Providers::Vmware::InfraManager::RefreshWorker"                  => [:vmware],
+  "ManageIQ::Providers::Vmware::NetworkManager::RefreshWorker"                => [:vmware],
+  "EmbeddedAnsibleWorker"                                                     => [],
+  "MiqCockpitWsWorker"                                                        => [:cockpit],
+  "MiqEmsMetricsProcessorWorker"                                              => [],
+  "MiqEmsRefreshCoreWorker"                                                   => [],
+  "MiqEventHandler"                                                           => [],
+  "MiqGenericWorker"                                                          => [],
+  "MiqPriorityWorker"                                                         => [],
+  "MiqReportingWorker"                                                        => [],
+  "MiqScheduleWorker"                                                         => [:scheduler],
+  "MiqSmartProxyWorker"                                                       => [:smartstate],
+  "MiqUiWorker"                                                               => %i(ui_dependencies web_server),
+  "MiqVimBrokerWorker"                                                        => [:manageiq_default],
+  "MiqWebServiceWorker"                                                       => %i(rest_api ui_dependencies web_server),
+  "MiqWebsocketWorker"                                                        => %i(ui_dependencies web_server web_socket)
+}.freeze
```

Comment:
👍 I approve of the sorting and agree with being consistent. (Although I prefer `[]` over `%i()`)